openpype/hosts/max/plugins/publish/validate_attributes.py

```diff
@@ -28,32 +66,24 @@ def get_invalid(cls, context):
         )
         if not attributes:
             return
+        invalid = []
+        for object_name, required_properties in attributes.items():
+            if not rt.Execute(f"isValidValue {object_name}"):
+                # Skip checking if the node does not
+                # exist in MaxWrapper Class
+                continue
```

Comment:
Should this log a warning or debug message? How likely is it that an admin might configure settings for something that is unavailable to an artist's machine? And if that's the case should the user be warned about that or should we 'not care' in general?

If it's artist important, make it a warning or info log.
If it's totally fine for it to not exist make it a debug log.

Without logs it'll be much harder to identify that the admin might have just mistyped in settings "renderer.prodction" and they'll be guessing at why it doesn't validate it.

Reply:
> or deleted 

Deleted? What's being deleted where?

Something like this would be somewhat informational at least:
```python
self.log.debug(f"Unable to find '{object_name}'. Skipping validation of attributes")
```

Reply:
> > or deleted
> 
> Deleted? What's being deleted where?
> 
> Something like this would be somewhat informational at least:
> 
> ```python
> self.log.debug(f"Unable to find '{object_name}'. Skipping validation of attributes")
> ```

Somehow the custom properties can be removed by users, and these can be bind to the MaxWrapper. In simplest words, the properties are still binding to the MaxWrapper(we can still check there is invalid object with MaxWrapper) but the value of these are missing.
Hopefully this explanation wont confuse you, as it somehow confuses me when I look at their [docs](https://help.autodesk.com/view/MAXDEV/2023/ENU/?guid=GUID-95453E22-A022-4543-B31C-A052CECD3598)